pkg/metrics/metrics.scss

```diff
@@ -551,16 +549,26 @@ $graphs: cpu, memory, disks, network;
     }
 
     .polygon.polygon-use {
-      transform: scale(-1);
+      transform: scaleX(-1);
```

Comment:
(Transfering from #18774) This is already broken on main, and this makes it worse. See https://github.com/cockpit-project/cockpit/pull/18774#pullrequestreview-1421254336 . Please cherry-pick https://github.com/cockpit-project/cockpit/pull/18774/commits/01a27e2aba6246e7f94fa6cf3f82bc1f9d3acda4 at the top of the stack (as it's relative to main) to fix the graphs. Thanks!

Reply:
I sent PR #18786 for that. Blocking this PR on that, as the former will need a pixel update.

Reply:
#18786 landed, unblocked.